src/app/workspaces/create-workspace/create-workspace.controller.js

```diff
@@ -50,8 +50,6 @@ class CreateWorkspaceCtrl {
     promise.then(() => {
       this.updateData();
     });
-
```

Comment:
you should remove rootScope from the constructor as it was introduced for this broadcast